build.cake

```diff
@@ -160,7 +108,7 @@ Task("BuildHelloWorld")
         var helloWorldProjectPathParam = " --project=\"./docker/integration_tests/XCProjects/HelloWorldV5/HelloWorldV5_Model.xcml\"";
     
         var mono = "mono";
-        var xcbuild = "./tools/XComponent.Build.Community/tools/XCBuild/xcbuild.exe";
+        var xcbuild = "./tools/xcomponent.build.community/6.0.3/tools/XCBuild/xcbuild.exe";
```

Comment:
The path should not depend on the version (perhaps the nuget install of xcomponent.build is wrong)

Reply:
it's download in the packages.csproj with :     <PackageReference Include="XComponent.Build.Community" Version="6.0.3"/>

and the path is cast with the version

Reply:
it's download in the 'tools/packages.csproj' with :

    <PackageReference Include="XComponent.Build.Community" Version="6.0.3"/>


and the path is cast with the version

Reply:
That's the reason why we do not add the reference of xcomponent.build in packages.config.
Instead, just use the previous configuration in cake:
Setup(context=> {
    DoInDirectory(@"tools", () => {
        NuGetInstall("XComponent.Build.Community", new NuGetInstallSettings{ Version=XComponentVersion, ExcludeVersion=true });
    });
});
Update this code with DotNetCoreRestore

Reply:
I add 'addin "Cake.XComponent&version=6.0.0"' to do like Koordinator for compile project HelloWorld